Website/DesktopModules/Hotcakes/Core/Admin/Parts/CreditCardGateways/PayPal Payments Pro/Edit.ascx

```diff
@@ -4,16 +4,12 @@
 </h1>
 <div class="hcForm">
     <div class="hcFormItemHor">
-        <asp:Label runat="server" resourcekey="Username" CssClass="hcLabel" />
-        <asp:TextBox ID="txtUsername" runat="server" />
+        <asp:Label runat="server" resourcekey="ClientId" CssClass="hcLabel" />
```

Comment:
In most places, there is a prefix to help the code inform developers what objects are. In this case, the `ClientId` should be `lblClientId`.  This update will also require updating the key in the requisite localization file.

Reply:
Done. Do you want that I modify this on the rest of the fields that were already present?

Reply:
It should only be done on the ones you've been editing.  We can edit the other ones later, as we continue to update everything. :)